x-pack/plugins/alerting/server/rules_client/lib/wait_before_next_retry.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// number of times to retry when conflicts occur
```

Comment:
consider removing it as it doesn't add any value

Reply:
1b07f410076e889147de3a82f9ed4d93909d2f7d